src/k2/module/k23si/Module.cpp

```diff
@@ -646,12 +646,12 @@ K23SIPartitionModule::handleQuery(dto::K23SIQueryRequest&& request, dto::K23SIQu
         request.key = key_it->first; // if we retry, do so with the key we're currently iterating on
         return _doPush(request.key, versions.WI->data.timestamp, request.mtr, deadline)
```

Comment:
I think maybe we shouldn't track numScans across pushes. The idea is to get a feel for how much the reactor is monopolized by the query, and the push will yield it. We can see what Ivan thinks.

Reply:
done.